src/components/views/right_panel/GroupHeaderButtons.js

```diff
@@ -66,8 +67,13 @@ export default class GroupHeaderButtons extends HeaderButtons {
     }
 
     _onMembersClicked() {
-        // This toggles for us, if needed
-        this.setPhase(RIGHT_PANEL_PHASES.GroupMemberList);
+        if (this.state.phase === RIGHT_PANEL_PHASES.GroupMemberInfo) {
+            // send the active phase to trigger a toggle
+            this.setPhase(RIGHT_PANEL_PHASES.GroupMemberInfo, RightPanelStore.getSharedInstance().roomPanelPhaseParams);
```

Comment:
Per: https://github.com/matrix-org/matrix-react-sdk/pull/3804/files#r366672560
```suggestion
            this.setPhase(RIGHT_PANEL_PHASES.GroupMemberInfo);
```

Reply:
If you insist